src/locales/pl/account.ftl

```diff
@@ -0,0 +1,61 @@
+### Localization for Account
+
+account-tokenNotFound =
+  Ten link nie prawidłowy, sprawdź, czy właściwie go wklejasz.
+
+## Password Reset
+
+resetPassword-resetPassword = Zmiana hasła
+resetPassword-resetYourPassword = Zmień swoje hasło
+resetPassword-pleaseEnterNewPassword =
+  Prosimy o podanie nowego hasła, żeby móc się logować na konto.
+  Upewnij się, że jest unikalne i dbaj o jego bezpieczeństwo.
+resetPassword-passwordLabel = Hasło
+resetPassword-passwordDescription = Musi mieć co najmniej {$minLength} znaków
+resetPassword-passwordTextField =
+  .placeholder = Hasło
+resetPassword-successfullyReset = Hasło zostało zmienione
+resetPassword-youMayClose =
+  Możesz już zamknąć to okno i zalogować się używają swojego nowego hasła. 
+resetPassword-oopsSorry = Przepraszamy!
+resetPassword-missingResetToken = Brakuje tokena do resetowania hasła!
+
+## Email Confirmation
+
+confirmEmail-emailConfirmation = Potwierdzenie emaila
+confirmEmail-confirmEmail = Potwierdź email
+confirmEmail-pleaseClickToConfirm = Kliknij poniżej, żeby potwierdzić email.
+confirmEmail-oopsSorry = Przepraszamy!
+confirmEmail-missingConfirmToken = Brakuje tokena do resetowania hasła.
+confirmEmail-successfullyConfirmed = Email został potwierdzony
+confirmEmail-youMayClose =
+  Możesz już zamknąć to okno.
+
+## Download
+
+download-landingPage-title = Pobierz swoją historię komentarzy
+download-landingPage-description =
+  Your comment history will be downloaded into a .zip file. After your
+  comment history is unzipped you will have a comma separated value (or .csv)
+  file that you can easily import into your favorite spreadsheet
+  application.
+download-landingPage-contentsDescription =
+  For each of your comments the following information is included:
+download-landingPage-contentsDate =
+  When you wrote the comment
+download-landingPage-contentsUrl =
+  The permalink URL for the comment
+download-landingPage-contentsText =
+  The comment text
+download-landingPage-contentsStoryUrl =
+  The URL on the article or story where the comment appears
+download-landingPage-downloadComments =
+  Download My Comment History
+download-landingPage-sorry = Your download link is invalid.
+
+## Unsubscribe
+
+unsubscribe-oopsSorry = Oops Sorry!
+unsubscribe-successfullyUnsubscribed = You are now unsubscribed from all notifications
+unsubscribe-clickToConfirm = Click below to confirm that you want to unsubscribe from all notifications.
+unsubscribe-confirm = Confirm
```

Comment:
Missed translations

Reply:
Tracking fix in CORL-936

Reply:
Created updated pull request (#2852) with missing phrases and updated numerals. Sorry and thank you for noticing.